src/lit-element.ts

```diff
@@ -126,6 +126,8 @@ export class LitElement extends UpdatingElement {
       (result: unknown, container: Element|DocumentFragment,
        options: ShadyRenderOptions) => void = render;
 
+  ssrRenderChildren: IterableIterator<string> | undefined;
```

Comment:
I have to review the other PRs, but docs would be good here and this is unused locally, so what's it do?